0.16.0-release-notes.md

```diff
@@ -11,6 +11,10 @@ This is the draft release notes for 0.16.0. It is scheduled to be released on
     Instead, the history listing mode now applies smart-case matching by
     default.
 
+-   Declaring a variable with a `~` suffix, without an explicit initial value,
+    now defaults to $builtin:nop rather than $builtin:nil
```

Comment:
$builtin:nop~

Also, since the `builtin:` namespace no longer exists without an explicit `use`, it's better to just say "builtin" without using it as a variable name qualifier; same for doc and comments below.